portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -15,16 +15,19 @@
 package com.google.sps.servlets;
 
 /** This class sets the text and date data for a {@code Comment} */
-public class Comment {
-  private long day;
-  private long month;
-  private long year;
-  private String text;
+public final class Comment {
 
-  public Comment(String text, long day, long month, long year) {
+  /** postedDateTIme should appear like this: Jun 11, 2020 11:18 AM */  
```

Comment:
Just FYI (not actionable): if/when we start to use a builder pattern with setters/getters - this is something you could verify when setting the field. And if the input param doesn't match the expected convention, then throw an error

Reply:
Oh okay, cool!